src/index.js

```diff
@@ -24,43 +27,47 @@ export class LSP6Signer {
       throw new Error("Private key must be 32 bytes long");
     }
 
-    var hash = this.hashMessage(data);
+    var hash = this.hashMessage(message);
     var signature = Account.sign(hash, privateKey);
     var vrs = Account.decodeSignature(signature);
-    return {
-      message: data,
+    var messageInfo = {
+      message: message,
       messageHash: hash,
       v: vrs[0],
       r: vrs[1],
       s: vrs[2],
       signature: signature,
     };
+    return messageInfo;
   }
 
-  recover(message, signature, preFixed) {
+  recover(message, signature, isMessagePrefixed) {
     var args = [].slice.apply(arguments);
+    // const args = arguments
 
     if (!!message && typeof message === "object") {
+      const messageInfo = message;
```

Comment:
I think we can just use the `message` variable here rather than creating the `messageInfo` variable